src/main/java/com/company/TAM2/Main.java

```diff
@@ -0,0 +1,47 @@
+package com.company.TAM2;
+
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Main {
+
+    public static final int COLUMNS_COUNT = 2;
+
+    public static void main(String[] args) {
+        String fromFileName =  "orange.csv";
+        String toFileName = "result.txt";
+        getStatistic(fromFileName, toFileName);
+    }
+
+    public static void getStatistic(String fromFileName, String toFileName) {
+        try (
+            BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+            BufferedWriter writer = new BufferedWriter(new FileWriter(toFileName));
+        ) {
+            Map<String, Integer> actionCount = new HashMap<>();
+            String line;
+
+            while ((line = reader.readLine()) != null){
+                String[] parts = line.split(",");
+                if(parts.length != COLUMNS_COUNT){
+                    throw new IllegalArgumentException("Not correct data");
+                }
+
+                String action = parts[0].trim();
+                int count = Integer.parseInt(parts[1].trim());
+
+                actionCount.put(action, count);
```

Comment:
doesn't we need an summirize amont by operation to have a correct result? I think just save the amount from the file line will not be enough

Reply:
fixed